client/src/containers/UserProfile/UserProfile.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { fetchEvents } from '../../actions';
+import Profile from './../../components/UserProfile/UserProfile';
+
+class UserProfile extends Component {
+  constructor(props) {
+    super(props);
+    this.fetchEvents();
+  }
+
+  fetchEvents() {
+    const { dispatch } = this.props;
+    dispatch(fetchEvents());
+  }
+  getUserDetails() {
```

Comment:
@isaadabbasi what about return type here? We should be adding them almost everywhere